### PR TITLE
Add jerry story

### PR DESCRIPTION
**Pull request by mooreds**
Jerry's community story.

### CODE REVIEW

File: site/_posts/2020-09-23-jerry-hopper-fusionauth.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: blog-post
-title: Contaim adopts FusionAuth to manage their auth lifecycle
-description: Contaim CEO Jerry Plunkett discusses how FusionAuth has helped his business.
+title: Jerry loved FusionAuth so much he built an Arm image for it
```
Comment (robotdan) [inline site/_posts/2020-09-23-jerry-hopper-fusionauth.md:3]: Arm -> ARM ?

I could only find `Arm` when referring to `Arm Holdings` but as an arch I think it is always `ARM`?

File: site/_posts/2020-09-23-jerry-hopper-fusionauth.md [hunk 2]
```diff
@@ -10,131 +10,109 @@ tags: topic-customer-story
 excerpt_separator: "<!--more-->"
 ---
 
-Jerry Hopper is a FusionAuth user and community member. He chatted with us over email about how he is using FusionAuth to meet his auth needs. 
+Jerry Hopper is a FusionAuth user and community member. He also maintains the Arm/Arm64 FusionAuth docker repo. He chatted with us over email about how he is using FusionAuth to meet his auth needs.
```
Comment (robotdan) [inline site/_posts/2020-09-23-jerry-hopper-fusionauth.md:13]: `arm/arm64` ?

Comment (mooreds) [reply site/_posts/2020-09-23-jerry-hopper-fusionauth.md]: nice catch!

File: site/_posts/2020-09-23-jerry-hopper-fusionauth.md (hunk 1 above)
Comment (mooreds) [reply site/_posts/2020-09-23-jerry-hopper-fusionauth.md]: nice catch!

File: site/_posts/2020-09-23-jerry-hopper-fusionauth.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+With the 'rise of ARM devices', thanks to the Raspberry Pi, I discovered that FusionAuth was able to run on arm/arm64. I'm maintaining the [FusionAuth docker repo for it](https://github.com/jerryhopper/fusionauth-app-multiarch). Since that time, a 1GB SBC powers my `oauth2_proxy`.
+
+**Dan:** What problems did FusionAuth solve for you? And how were you solving them before FusionAuth?
+
+**Jerry:** The best thing about Fusionauth is that I never have any headaches about authentication mechanisms. In my pre-FusionAuth era, I was more or less dependent on services like 'Login with Google/Facebook'. This doesn't work well when you are developing in places where there is no network.
+
+Another nice aspect is that FusionAuth is API driven, and has very complete logging, which makes it a breeze to make your applications GDPR compliant.  
+
+The fact that fusionauth supports multiple tenants on one instance, helps when developing several different projects.
```
Comment (dpolstra) [inline site/_posts/2020-09-23-jerry-hopper-fusionauth.md:46]: "The fact that fusionauth" -> FusionAuth

File: site/_posts/2020-09-23-jerry-hopper-fusionauth.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+Meanwhile the HRMatches application development was on hold, as the client wasn't paying its bills. After three months the HRMatches project came to a halt as the client went bankrupt. Sad detail: just a few months before bankruptcy we convinced the boss to ditch Apereo/CAS for FusionAuth. But because of the bankruptcy we never had the chance to implement it for HRMatches.
+
+**Dan:** Can you talk a bit more about how you use FusionAuth to make your projects GDPR compliant?
+
+**Jerry:** We compiled a list with minimal needs for the auth service that at least 'show your intentions to follow GDPR'. (Yes, intentions were very much a thing when GDPR was introduced and still are. Proof of good intentions can save you money when GDPR violations are judged or fined.)
+
+> Obviously a lot of GDPR compliance needs to come from the custom code in the application we build, but FusionAuth's API allowed us to do this with ease. Having the authentication separate from the main application gives us the flexibility to adapt.
+
+Our GDPR checklist for an auth service:
```
Comment (dpolstra) [inline site/_posts/2020-09-23-jerry-hopper-fusionauth.md:74]: This is what our GDPR checklist for an auth service looks like:

File: site/_posts/2020-09-23-jerry-hopper-fusionauth.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+ 
+But I was both a developer as well as technical support for some clients, for whom I needed to use a VPN. And when I was on the VPN my colleagues couldn't reach my virtual FusionAuth instance. My boss wouldn't allow me to run FusionAuth on the dev servers, so I just plugged a NanoPi in the network so my colleagues could play with it, even when I wasn't there.  It was my way of trying to convince my colleagues and giving the application maximum exposure. I think it was after one to two months I decided to deploy an arm64 device with FusionAuth in the office as I wasn't allowed to run a dedicated one on development servers. I do think that the web-interface and 'local availability' of the FusionAuth instance was the key to convincing my colleagues that FusionAuth was very suitable for us.
+
+**Dan:** Any general feedback/areas to improve?
+
+**Jerry:** I think for completeness sake the 'Client Credentials' grant should be implemented. [ed note: see [this issue](https://github.com/FusionAuth/fusionauth-issues/issues/155) for more on that]
+
+Oh, and would love to see official multi-arch docker builds ;).
```
Comment (dpolstra) [inline site/_posts/2020-09-23-jerry-hopper-fusionauth.md:112]: Oh, and **I** would love to see official multi-arch docker builds ;).

File: site/_posts/2020-09-23-jerry-hopper-fusionauth.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
+---
+layout: blog-post
+title: Jerry loved FusionAuth so much he built an ARM image for it
```
Comment (dpolstra) [inline site/_posts/2020-09-23-jerry-hopper-fusionauth.md:3]: Jerry riffs on FusionAuth GDPR compliance and that time he built an ARM image for it

File: site/_posts/2020-09-23-jerry-hopper-fusionauth.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+description: Community member Jerry Hopper discusses how and why he sold FusionAuth to his colleagues.
```
Comment (dpolstra) [inline site/_posts/2020-09-23-jerry-hopper-fusionauth.md:4]: Community member Jerry Hopper discusses how he discovered FusionAuth and how it's helped him and his teams save development time and authentication headaches.

**Pull request merged**